training/sklearn/structured/base/setup.py

```diff
@@ -19,7 +19,7 @@
 
 REQUIRED_PACKAGES = [
     'tensorflow==1.13.1',
-    'scikit-learn>=0.19.1',
+    'scikit-learn>=0.20.3',
```

Comment:
CMLE does not yet support this version. RUNTIME 1.13 only supports 0.20.2: https://cloud.google.com/ml-engine/docs/tensorflow/runtime-version-list